.vscode/settings.json

```diff
@@ -0,0 +1,23 @@
+{ 
+	"intelephense.files.exclude": [
+	"**/.git/**",
+	"**/.svn/**",
+	"**/.hg/**",
+	"**/CVS/**",
+	"**/.DS_Store/**",
+	"**/node_modules/**",
+	"**/bower_components/**",
+	"**/vendor/**/{Tests,tests}/**",
+	"**/.history/**",
+	"**/vendor/**/vendor/**",
+	"**/packages/**/vendor/**",
+	"**/projects/**/vendor/**",
+	"**/plugins/**/vendor/**",
```

Comment:
At this point it seems like you might just go ahead and list `"**/vendor/**"`.

That would also make lines 10 and 12 redundant.